crates/proof-of-sql/Cargo.toml

```diff
@@ -75,6 +75,7 @@ development = ["arrow-csv"]
 default = ["arrow", "blitzar", "rayon"]
 arrow = ["dep:arrow"]
 test = ["dep:rand"]
+std = ["snafu/std"]
```

Comment:
This is causing CI to fail. (Take a look if you're curious.)
You _could_ fix the CI here, but I would much rather simply you add `features = ["std"]` to `snafu`.

The reason for this is that I have a PR incoming that adds true `no_std` support and will be adding a `std` feature. It would be much cleaner to add this feature there, rather than here.

I'm now aware of the `posql_db` concern and the `snafu/std` feature. (I may just disable the `clap/std` feature, for example.)

Reply:
That makes completely sense. I've addressed this in https://github.com/spaceandtimelabs/sxt-proof-of-sql/pull/181/commits/31910e5b098e94646f1087cfb2910590933556f0